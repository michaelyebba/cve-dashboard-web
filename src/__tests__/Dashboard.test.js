import React from 'react';
import DashboardPage from '../components/pages/Dashboard';
import {HashRouter} from 'react-router-dom'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
        <HashRouter>
            <DashboardPage />
        </HashRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});