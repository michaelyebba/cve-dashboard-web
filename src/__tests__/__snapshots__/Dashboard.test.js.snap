// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    id="header"
  >
    <div
      className="container"
    >
      <nav>
        <a
          className="logo"
          href="#/"
          onClick={[Function]}
        >
          <i
            className="fas fa-exclamation-triangle"
          />
          TOP 10 CVE Dashboard
        </a>
        <ul
          className="reports"
        >
          <li>
            <a
              aria-current="page"
              className="active"
              href="#/"
              onClick={[Function]}
              style={Object {}}
            >
              ALL
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/HIGH"
              onClick={[Function]}
            >
              HIGH
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/MEDIUM"
              onClick={[Function]}
            >
              MEDIUM
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/LOW"
              onClick={[Function]}
            >
              LOW
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <div
    className="container"
  >
    <nav
      className="navbar navbar-light bg-light"
    >
      <form
        className="form-inline"
      >
        <input
          className="form-control mr-sm-2"
          onChange={[Function]}
          placeholder="Search by product"
          type="search"
          value=""
        />
        <input
          className="btn btn-outline-success my-2 my-sm-0"
          onClick={[Function]}
          type="button"
          value="Search"
        />
      </form>
    </nav>
  </div>
  <p
    style={
      Object {
        "margin": "auto",
        "maxWidth": "1000px",
        "padding": "20px 20px",
      }
    }
  >
    Loading, please wait...
  </p>
  <footer
    id="footer"
  >
    <div
      className="container"
    >
      <p>
        <small>
          Common Vulnerabilities and Exposures (CVE) is a catalog of known security threats. The catalog is sponsored by the United States Department of Homeland Security (DHS). 
        </small>
      </p>
    </div>
  </footer>
</div>
`;
